airflow/api_connexion/openapi/v1.yaml

```diff
@@ -3439,9 +3443,26 @@ components:
           type: boolean
           default: true
 
+        tasks:
+          description: |
+            A list of {task_id, map_index} combinations to clear.
```

Comment:
Is it worth mentioning (somewhere? In the example? Is that possible) that you can pass both `tasks` and `task_ids` at the same time to this endpoint?

Reply:
Is this meant to be communicated to the users? If not, YAML can contain comments.

Reply:
Yes, was thinking about communicating this to users via the rendered API docs, on https://airflow.apache.org/docs/apache-airflow/stable/stable-rest-api-ref.html